Tools/ardupilotwaf/boards.py

```diff
@@ -378,6 +379,13 @@ def get_board(ctx):
 # be worthy to keep board definitions in files of their own.
 
 class sitl(Board):
+
+    def __init__(self):
+        if Utils.unversioned_sys_platform().startswith("linux"):
```

Comment:
We don't have cfg.env available in init, so I have moved the setting to configure_env

Reply:
I am not going to resolve this here, as DEST_OS is not available under init, and not setting with_can under __init__ breaks builds. I believe it will involve larger change on how with_can is declared and used, I want to this in AP_Periph PR, as it will be needing change there anyways.